### PR TITLE
[Privatization] Register MakeOnlyUsedByChildrenProtectedRector to privatization config set

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

File: packages/post-rector/src/Collector/UseNodesToAddCollector.php [hunk 1]
```diff
@@ -174,7 +174,7 @@ public function getShortUsesByFileInfo(SmartFileInfo $smartFileInfo): array
     /**
      * @return AliasedObjectType[]|FullyQualifiedObjectType[]
      */
-    public function getUseImportTypesByNode(Node $node): array
+    private function getUseImportTypesByNode(Node $node): array
```
Comment (samsonasik) [inline packages/post-rector/src/Collector/UseNodesToAddCollector.php:177]: it is strange that it changed to `private`, as the rule only change to `protected`, it seems the re-use of `getChildrenOfClass()` make it not passed.

Comment (samsonasik) [reply packages/post-rector/src/Collector/UseNodesToAddCollector.php]: fixed by skip final class

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

**Comment by TomasVotruba** [comment]
Thank you :+1:

**Pull request merged**